### DISCUSSION TITLE
Add device class "identify" for button entities

### DISCUSSION THREAD

**Discussion by frenck** [home-assistant/architecture#934] [Entity Models] [answered]
## Context

We have quite a few integrations that expose a "identify" button for their devices.

This is for example used to identify the bulb (which will blink one or twice when the button is pressed).

Integrations currently providing some sort of identify button entities for their devices:

- `elgato`
- `homekit_controller`
- `homewizard`
- `hunterdouglas_powerview`
- `lifx`
- `overkiz`
- `zha`

## Proposal

To add the `identify` as a device class to the `ButtonDeviceClass` enum, being `ButtonDeviceClass.IDENTIFY`.

The default entity name for the device class will be set to "Identify".

The icon to use in the frontend by default:

![CleanShot 2023-06-22 at 23 31 45](https://github.com/home-assistant/architecture/assets/195327/f7eeffc0-f883-4856-aa8f-f8c077e9768e)

**Comment by balloob** [comment, marked as answer]
Sounds good.

**Comment by frenck** [comment]
Implementing this in:

- Core: <https://github.com/home-assistant/core/pull/95244>
- Frontend: <https://github.com/home-assistant/frontend/pull/17036>
- Developer documentation: <https://github.com/home-assistant/developers.home-assistant/pull/1818>
- End-user documentation: https://github.com/home-assistant/home-assistant.io/pull/27971
